### PR TITLE
add checklist

### PR DESCRIPTION
**Pull request by santiagonoguez**
## Description
Add a simple test checklist to request template

## Motivation and Context
As specified as action on https://docs.google.com/document/d/1T2NQ8S79Yz-VGu-wq9fePT-DuoiGb9K2ULOCxdXqtmw/edit#heading=h.gqvdnto8v52o

## How Has This Been Tested?
No need to test, it's just a template update.

## Release Plan:
Won't be released to production, just merged to master.

#### Release Checklist Items Skipped?
N/A

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,8 +7,11 @@ Why is this change required? What problem does it solve?
 ## How Has This Been Tested?
 Describe in detail how you tested your changes. Include details of your testing environment and link(s) for reviewers to validate.
 
+ - [ ] Common template has been tested on Player
+ - [ ] Common template has been tested on Apps
```
Comment (stulees) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: @santiagonoguez I think Apps is a bit vague. Can we say _Common template has been tested in Apps Template Editor_?

Comment (santiagonoguez) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: @stulees of course, I'll change the wording, thanks !

Comment (santiagonoguez) [reply .github/PULL_REQUEST_TEMPLATE.md]: @stulees changed !

**Review by stulees — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (stulees) [reply .github/PULL_REQUEST_TEMPLATE.md]: @santiagoarriaga thanks!

**Pull request merged**